source/includes/fundamentals/explain-verbosity.rst

```diff
@@ -0,0 +1,23 @@
+The following table shows the verbosity levels for the explain
+operation and their intended use cases:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 40 60
+
+   * - Verbosity Level
+     - Use Case
+
+   * - ALL_PLANS_EXECUTIONS
+     - You want to know which plan MongoDB will choose to run your query.
+
+   * - EXECUTION_STATS
+     - You want to know if your query is performing well.
+
+   * - QUERY_PLANNER
+     - You have a problem with your query and you want as much information
+     as possible to diagnose why it might be slow.
+     
+For information on what each verbosity level does, see the API
+Documentation on :java-core-api:`ExplainVerbosity <com/mongodb/ExplainVerbosity>`.
```

Comment:
```suggestion
Documentation on the :java-core-api:`ExplainVerbosity <com/mongodb/ExplainVerbosity>` Enum.
```

Suggestion: Could the type of this item be mentioned? I think it is an Enum. I haven't described anything with this type before, so some research may have to be done to see how people writing about Java talk about Enums.

Reply:
My edit shows "enumeration" instead of "Enum"